terraform/e2e/main.tf

```diff
@@ -59,3 +59,34 @@ module "policy_service" {
   oidc_service_account = module.common_infra.oidc_service_account
   gcs_events_filter    = var.policy_gcs_events_filter
 }
+
+resource "random_id" "default" {
+  byte_length = 2
+}
+
+locals {
+  dev_static_bucket_name = "pmap-static-dev-bucket-${random_id.default.hex}"
```

Comment:
use `static_bucket_name        = "pmap-static-ci-bucket-${random_id.default.hex}"`

Reply:
Done.

Reply:
You are still using the previous variable name.

Reply:
Done.